src/test/ui/feature-gates/feature-gate-static-nobundle.rs

```diff
@@ -1,5 +1,8 @@
+// ignore-tidy-linelength
+
```

Comment:
```suggestion
```

`//~ WARNING` lines and similar are exempt from the linelength check now that https://github.com/rust-lang/rust/pull/83549 has landed.